examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -114,13 +118,38 @@ class WebGPURenderer {
 
 			const renderTargetProperties = this._properties.get( renderTarget );
 
-			colorAttachment.attachment = renderTargetProperties.colorTextureGPU.createView();
-			depthStencilAttachment.attachment = renderTargetProperties.depthTextureGPU.createView();
+			if ( this._antialias ) {
+
+				colorAttachment.attachment = renderTargetProperties.colorBuffer.createView();
+				colorAttachment.resolveTarget = renderTargetProperties.colorTextureGPU.createView();
+				colorAttachment.storeOp = GPUStoreOp.Store;
```

Comment:
This should be GPUStoreOp.Clear, unless you're planning on reusing the content of the multisampled texture. It helps a TON with memory bandwidth on tiler GPUs. The resolve target pixels will be stored, but multisampled pixels  will be forgotten and the texture marked for lazy-clearing internally in Dawn/Chromium.

Reply:
Thanks for the comment. Updated.

Reply:
@takahirox I had to remove the modifications to `storeOp` since enabling antialiasing produced a red screen. Would be great if you can revisit this setting: 8e70942d186d761d892a9307a39faddb1c28172f

Reply:
Can you share a demo or code snippet which causes the problem? Here I enabled antialias in webgpu sandbox example but couldn't reproduce it.

Reply:
Strange, it does not work on my system. I just add the code back and set `antialias` to `true` in the example.

Reply:
Hm it could be platform or hardware specific issue? Working fine on my Windows10 + Canary(87.0.4257.0) + intel(R) HD graphics 530.

Anyways, using default for now is ok to me because it still works on my system, too. Let's revisit later if needed.

Reply:
Okay, I'll try to check the demo tomorrow on a Windows 10 notebook.